Makefile

```diff
@@ -139,6 +139,8 @@ clean:
 	rm -rf /tmp/avocado*
 	find . -name '*.pyc' -delete
 	find $(AVOCADO_OPTIONAL_PLUGINS) -name '*.egg-info' -exec rm -r {} +
+	# Remove this after 36lts is declared EOL
+	rm -rf avocado.egg-info
```

Comment:
OK, to be fair removing all eggs could surprise someone... Even this is not really kosher, but it is useful...